terminus-ui/drawer/src/drawer.component.scss

```diff
@@ -0,0 +1,115 @@
+@import './../../scss/helpers/animation';
+@import './../../scss/helpers/reset';
+@import './../../scss/helpers/color';
+@import './../../scss/helpers/cursors';
+@import './../../scss/helpers/shadows';
+@import './../../scss/helpers/spacing';
+@import './../../scss/helpers/typography';
+
+$ts-drawer-content-z-index: 1;
+$ts-drawer-push-drawer-z-index: 2;
+$ts-drawer-backdrop-z-index: 3;
+$ts-drawer-over-drawer-z-index: 4;
+
+// Mixin that creates a new stacking context.
+@mixin ts-drawer-stacking-context($z-index: 1) {
+  position: relative;
+  // Use a z-index to create a new stacking context. (We can't use transform because it breaks fixed
+  // positioning inside of the transformed element).
+  z-index: $z-index;
+}
+
+@mixin ts-fill {
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.ts-drawer-container {
+  // We need a stacking context here so that the backdrop and drawers are clipped to the
+  // TsDrawerContainer. This creates a new z-index stack so we use low numbered z-indices.
+  // We create another stacking context in the '.ts-drawer-content' and in each drawer so that
+  // the application content does not get messed up with our own CSS.
+  @include ts-drawer-stacking-context;
+  box-sizing: border-box;
+  // Need this to take up space in the layout.
+  display: block;
+  -webkit-overflow-scrolling: touch;
+  &[fullscreen] {
```

Comment:
I think we can lose it.. since our drawers can be inside any container `fullscreen` likely wouldn't actually be fullscreen anyway. If product ends up requesting the ability to fill the container we can add an `@Input` etc

Reply:
☝️